source/wp-content/themes/wporg-developer-2023/patterns/cli-commands.php

```diff
@@ -7,17 +7,17 @@
 
 ?>
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space","bottom":"var:preset|spacing|60"}}},"className":"alignfull","layout":{"type":"constrained","wideSize":"1280px","contentSize":"680px"}} -->
-<main class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--edge-space)">
-
-	<!-- wp:group {"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|20"}}},"layout":{"type":"constrained","justifyContent":"left"}} -->
-	<div class="wp-block-group alignwide" style="margin-top:var(--wp--preset--spacing--20)">
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|edge-space","left":"var:preset|spacing|edge-space"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--edge-space);padding-left:var(--wp--preset--spacing--edge-space)">
```

Comment:
Looks like this removed some spacing at the footer:

![Screen Shot 2024-02-15 at 13 07 46](https://github.com/WordPress/wporg-developer/assets/541093/ff69d9b8-1393-473e-8b5c-d91b4fd619aa)

Reply:
Fixed in [643bc25](https://github.com/WordPress/wporg-developer/pull/499/commits/643bc254caa45e79609ecca1d11d0d01c554f84d)